package.json

```diff
@@ -165,6 +168,9 @@
     "setupFilesAfterEnv": [
       "./src/test/setup.js"
     ],
+    "globals": {
+      "AVAILABLE_STAGING_LOCALES": null
+    },
```

Comment:
Additionally, this was needed for the tests.

Reply:
I wonder if we can make the story better for tests, because with this we can't test the code when it's not null...
Do you have an idea? This can also come later I think.